src/librustc_mir/interpret/eval_context.rs

```diff
@@ -827,3 +831,21 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         frames
     }
 }
+
+impl<'ctx, 'mir, 'tcx, Tag, Extra> HashStable<StableHashingContext<'ctx>>
```

Comment:
Where did this impl come from?

Reply:
I expanded the macro because I planned to remove it in a subsequent commit. I can revert this.

Reply:
Yeah, maybe just revert it from this PR and leave it for later.